src/features/activeTodo.ts

```diff
@@ -0,0 +1,27 @@
+import { Todo } from '../types/Todo';
+
+type AddTodo = { type: 'todo/ADD', arr: Todo };
+type RemoveTodo = { type : 'todo/REMOVE' };
+
+const addTodo = (todo: Todo): AddTodo => ({ type: 'todo/ADD', arr: todo });
```

Comment:
`arr` is bad variable name

Reply:
still not resolved,
![image](https://github.com/mate-academy/react_redux-list-of-todos/assets/55200530/09c941a8-9baf-4505-8bc0-3f2641d893e3)